copier.yml

```diff
@@ -6,15 +6,15 @@ package_name:
   help: Enter the name of the Python package.
   validator: >-
         {% if not (package_name | regex_search('^[a-z][a-z0-9\_]+$')) %}
-        package_name must start with a letter, followed one or more letters, digits or underscores all lowercase.
+        package_name must start with a letter, followed one or more letters, digits or underscores all lowercase. Avoid using spaces or uppercase letters for the best experience across operating systems.
```

Comment:
Do we want to make this consistent with the README by also specifying to avoid dashes, i.e., 'Avoid using spaces, **dashes** or uppercase letters [...]'?

Reply:
This came from the README :)

Reply:
But good point!

Reply:
This is a restriction, as user is not **allowed** to use any of that. Shall we get rid of it altogher?

Reply:
The `directory_name` explanation indeed did not mention dashes, I was looking at the explanation for `package_name`, which does suggest avoiding them. Either way, I agree that removing it even better since it's not allowed anyway :+1:

Reply:
Thanks for your review @sjvrijn. I have made changes and updated branch. This can be merged now.